system/include/webgl/webgl1_ext.h

```diff
@@ -90,61 +128,97 @@ WEBGL_APICALL void GL_APIENTRY emscripten_webgl_getTranslatedShaderSource(GLuint
 #define GL_MAX_DRAW_BUFFERS_WEBGL 0x8824
 WEBGL_APICALL void GL_APIENTRY emscripten_glDrawBuffersWEBGL(GLsizei n, const GLenum *buffers);
 WEBGL_APICALL void GL_APIENTRY glDrawBuffersWEBGL(GLsizei n, const GLenum *buffers);
+#endif /* EMSCRIPTEN_WEBGL_draw_buffers */
 
 // 19. https://www.khronos.org/registry/webgl/extensions/ANGLE_instanced_arrays/
+#ifndef EMSCRIPTEN_ANGLE_instanced_arrays
+#define EMSCRIPTEN_ANGLE_instanced_arrays 1
 #define GL_VERTEX_ATTRIB_ARRAY_DIVISOR_ANGLE 0x88FE
 WEBGL_APICALL void GL_APIENTRY emscripten_glDrawArraysInstancedANGLE(GLenum mode, GLint first, GLsizei count, GLsizei primcount);
 WEBGL_APICALL void GL_APIENTRY emscripten_glDrawElementsInstancedANGLE(GLenum mode, GLsizei count, GLenum type, GLintptr offset, GLsizei primcount);
 WEBGL_APICALL void GL_APIENTRY emscripten_glVertexAttribDivisorANGLE(GLuint index, GLuint divisor);
 WEBGL_APICALL void GL_APIENTRY glDrawArraysInstancedANGLE(GLenum mode, GLint first, GLsizei count, GLsizei primcount);
 WEBGL_APICALL void GL_APIENTRY glDrawElementsInstancedANGLE(GLenum mode, GLsizei count, GLenum type, GLintptr offset, GLsizei primcount);
 WEBGL_APICALL void GL_APIENTRY glVertexAttribDivisorANGLE(GLuint index, GLuint divisor);
+#endif /* EMSCRIPTEN_ANGLE_instanced_arrays */
 
 // 20. https://www.khronos.org/registry/webgl/extensions/OES_texture_float_linear/
 // <no symbols exposed>
+#ifndef EMSCRIPTEN_OES_texture_float_linear
+#define EMSCRIPTEN_OES_texture_float_linear 1
+#endif /* EMSCRIPTEN_OES_texture_float_linear */
 
 // 21. https://www.khronos.org/registry/webgl/extensions/OES_texture_half_float_linear/
 // <no symbols exposed>
+#ifndef EMSCRIPTEN_OES_texture_half_float_linear
+#define EMSCRIPTEN_OES_texture_half_float_linear 1
+#endif /* EMSCRIPTEN_OES_texture_half_float_linear */
 
 // 25. https://www.khronos.org/registry/webgl/extensions/EXT_blend_minmax/
+#ifndef EMSCRIPTEN_EXT_blend_minmax
+#define EMSCRIPTEN_EXT_blend_minmax 1
 #define GL_MIN_EXT 0x8007
 #define GL_MAX_EXT 0x8008
+#endif /* EMSCRIPTEN_EXT_blend_minmax */
 
 // 27. https://www.khronos.org/registry/webgl/extensions/EXT_shader_texture_lod/
 // <no symbols exposed>
+#ifndef EMSCRIPTEN_EXT_shader_texture_lod
+#define EMSCRIPTEN_EXT_shader_texture_lod 1
+#endif /* EMSCRIPTEN_EXT_shader_texture_lod */
 
 // 12. https://www.khronos.org/registry/webgl/extensions/WEBGL_compressed_texture_atc/
```

Comment:
Unrelated, but this extension has been rejected as obsolete, so we should probably take it out.

Reply:
I think you removed the wrong one accidentally

Reply:
😓 WEBGL_compressed_texture_atc is obsolete. removed